src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1,4 +1,27 @@
-// Add the required types and props
-export const TodoInfo = () => (
-  <>TodoInfo markup</>
-);
+import React from 'react';
+import getClassName from 'classnames';
+
+import { UserInfo } from '../UserInfo';
+import { Todo } from '../../types/Todo';
+
+import './TodoInfo.scss';
+
+type Props = {
+  todo: Todo;
+};
+
+export const TodoInfo: React.FC<Props> = ({ todo }) => {
+  const { title, completed, user } = todo;
```

Comment:
```suggestion
  const { 
    title, 
    completed, 
    user 
  } = todo;
```
Provide newline for each prop if their amount is >= 3